index.bs

```diff
@@ -806,6 +811,33 @@ series of steps |steps|, run these steps:
 
 </div>
 
+## Session termination not initiated by the client ## {#web-transport-termination}
+
+<div algorithm="termination-initiated-by-server">
+Whenever a [=WebTransport session=] which is associated with a {{WebTransport}} |transport| is
+[=session/terminated=], run these steps:
+
+1. Let |cleanly| be a boolean representing whether the HTTP/3 stream associated with the CONNECT
+   request that initiated |transport|'s [=[[Session]]=] is in the "Data Recvd" state. [[!QUIC]]
+1. [=Queue a network task=] with |transport| to run these steps:
+  1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`, abort these steps.
+  1. Let |reason| be TBD.
+  1. Let |error| be TBD.
+  1. [=Cleanup=] |transport| with |reason|, |error| and the negation of |cleanly|.
+
+</div>
+
+<div algorithm="termination-caused-by-connection-error">
+Whenever a [=connection=] associated with a {{WebTransport}} |transport| gets a connection error,
```

Comment:
This can only happen once? Are all connection errors terminal?

Reply:
Will these utilize `WebTransportError`?

Reply:
> This can only happen once? Are all connection errors terminal?

This is the same question as https://github.com/w3c/webtransport/pull/281#discussion_r651703004.

> Will these utilize WebTransportError?

The connection error is not script accessible. `error` will be a `WebTransportError` (or one of its friends).